docs/meshstack.azure-developer-platform-quickstart-guide.md

```diff
@@ -0,0 +1,240 @@
+---
+id: meshstack.aks-developer-platform-quickstart-guide
+title: AKS Developer Platform Guide
+---
+
+## Introduction
+
+Welcome to the **Azure Developer Platform Guide**!
+This guide provides step-by-step instructions to help you set up your developer platform experience
+incorporating Azure Subscriptions, GitHub & GitHub Actions to deliver an easy-to-use starter kit to your application teams.
+
+## Goals
+
+By the end of this guide, you will have built a complete starter kit for your users that will provide:
+
+- A new GitHub repository
+- A meshStack project with:
+  - a Azure subscription
+  - a GitHub actions workflow with credentials to the Azure subscription
+
+## Prerequisites
+
+Before starting, ensure you have the following:
+
+- You have a basic understanding of building blocks in meshStack.
+  - *If you don’t, we recommend running through the [AWS S3 guide](./meshstack.building-aws-quickstart-guide.md) to learn how building blocks work.*
+- You have integrated Azure as a platform in meshStack.
+  - *If you don’t, we recommend running through the [Platform Setup guide](./meshstack.how-to.integrate-meshplatform.md) where you are guided to create an Azure platform in meshStack*
+
+## What Will We Do in This Guide?
+
+Shortly put, we will set up the following in this guide:
+
+- A building block definition that allows application teams to provision new GitHub repositories.
+- A building block definition that adds a GitHub Action workflow with the right credentials for the Azure subscription so application teams have an out-of-the-box working Azure CI/CD.
+- A building block definition that combines all these elements & more into a single packaged starter kit.
+
+## 1. GitHub Repository
+
+### Preparing GitHub Repository
+
+<-- Internal note: this part is completely copied from the AKS Dev Platform guide -->
+
+The GitHub Repository building block definition needs access to your GitHub organization.
+Follow the following steps to create a GitHub App that will be used to manage the repositories.
+
+1. [**Register a GitHub App**](https://docs.github.com/en/apps/creating-github-apps/registering-a-github-app/registering-a-github-app)
+    1. Make sure you give it sufficient permissions. When creating the app you’ll see a section “Repository permissions”. Under here make sure that you set the following:
+        1. “Administration” is set to “read & write”.
```

Comment:
Numbering is strange here. How about ether use 1. and 1.1, 1.2, 2. etc or dashes for the second hierarchy level